tests/gui/steps/common_steps.py

```diff
@@ -1,13 +1,14 @@
 from behave import given, step
-from main.trello.gui.page_objects.login_page import LoginPage
 
 
-@given('the user goes to "{page}"')
+@given('the user goes to "{page}" page')
 def step_impl(context, page):
-    context.driver.get(context.base_url + page)
+    context.login_page = context.page_factory.get_page(page)(context.driver)
+    if page == "login":
+        context.driver.get(context.base_url + '/login')
```

Comment:
not sure if this is better, you could have left just "/login", no need to say "login page"